.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -0,0 +1,163 @@
+        5. Keep the title short and descriptive. The title is not the place to write down a full description of the issue.
+        6. When deciding to leave out information in a field, leave it blank and empty. Avoid writing things such as `n/a` for empty fields.
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Description of the bug
+      description: Please provide a detailed description on the bug you encountered, in a readable and comprehensible way.
+      placeholder: |
+        After upgrading to version x.y.z of OpenBullet2, the "Wordlists" isn't showing anymore file. After testing with different users, none of them could see the "Wordlists". It might be a regression from pull request x. I have tried rebooting my host as well as my container multiple times. I tested this functionality on different clients, and it happens to all the tested clients (client x, y, z), I verified that there is a file located inside the "Wordlists" folder. This makes me believe it is a server side issue.
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: Reproduction steps
+      description: Reproduction steps should be complete and self-contained. Anyone can reproduce this issue by following these steps. Furthermore, the steps should be clear and easy to follow.
+      placeholder: |
+        1. Sign in on the OpenBullet2 web client, with an account, using a browser of your choice.
+        2. Navigate to Wordlists.
+        3. Select "Add from local file or Upload (remote)".
+        4. Enter a name for the Wordlist and complete other fields.
+        5. Add the Wordlist.
+        6. The file is in a folder and accessible by editor but can't be accessed from OpenBullet2.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: What is the current _bug_ behavior?
+      description: Write down the incorrect behavior that currently happens after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen has the server name appended to the text. The text shown is: "I am a cool disclaimer!".
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: What is the expected _correct_ behavior?
+      description: Write down the correct expected behavior that is supposed to happen after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen should only show the configured text. The text that should be shown is: "I am a cool disclaimer!".
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version of the software
+      description: What version of OpenBullet2 are you using?
+      options:
+        - 0.3.0
+        - 0.2.5
+        - Older*
```

Comment:
I wouldn't model this as a dropdown, or we would have to update it on each new version (and the dropdown would become very long over time). Instead, I would put an example in the description like `0.3.0` (or a placeholder).

Additionally, we're missing a dropdown to select between the web and the native client since there are two different possibilities when people report a bug.